client/gutenberg/extensions/tiled-gallery/layout/index.js

```diff
@@ -0,0 +1,73 @@
+/**
+ * External dependencies
+ */
+import photon from 'photon';
+import { __, sprintf } from '@wordpress/i18n';
+import { Component } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Square from './square';
+import GalleryImage from '../gallery-image';
+
+export default class Layout extends Component {
+	photonize( { height, width, url } ) {
```

Comment:
We never get `width` and `height` here: adding them to `pickRelevantMediaFiles` fixes it:

```js
imageProps.width = get( image, [ 'sizes', 'large', 'width' ] );
imageProps.height = get( image, [ 'sizes', 'large', 'height' ] );
```

in

https://github.com/Automattic/wp-calypso/blob/0850165273bd7ee74a19f676a0333a70a9b48415/client/gutenberg/extensions/tiled-gallery/edit.jsx#L43-L50

Reply:
You _do_ get them on already uploaded images. I suspect your suggestion will fix this todo, nice spot!

>  Get square-ish layouts working on upload

Reply:
There are more tricky things going on here, I'm going to hold off on any changes for now.